ReactiveCocoa/NSObject+Intercepting.swift

```diff
@@ -42,10 +45,12 @@ extension Reactive where Base: NSObject {
 	///
 	/// - parameters:
 	///   - selector: The selector to observe.
-	///
+	///   - includeReturnValue:  the array will include the value returned by the invocation.  Ignored if the selector returns Void.
 	/// - returns: A signal that sends an array of bridged arguments.
-	public func signal(for selector: Selector) -> Signal<[Any?], NoError> {
-		return base.intercept(selector).map(unpackInvocation)
+	public func signal(for selector: Selector, includeReturnValue: Bool = false) -> Signal<[Any?], NoError> {
```

Comment:
This is API-compatible but binary-incompatible change. So the original API should be kept as is and the new API should be added as a separete one.